code/frameworks/sveltekit/README.md

```diff
@@ -79,11 +79,11 @@ Install the framework:
 yarn add -D @storybook/sveltekit@next
 ```
 
-Update your `main.cjs` to change the framework property:
+Update your `main.js` to change the framework property:
 
```

Comment:
@yannbf, to keep us honest, as you already updated the filename here from `main.cjs` to `main.js`, it would be great if the line below matches the naming structure as well. Turning this into:
If you have a `svelteOptions` property in `.storybook/main.js` you need to remove that."

What do you think?